tox.ini

```diff
@@ -15,11 +15,12 @@ setenv=
 
 basepython=
     python3.8: python3.8
-    python3.12: python3.12
+    python3.13: python3.13
 
 deps=
-    django4.2: Django>=4.2,<4.3
-    django5.0: Django>=5.0,<5.1
+    django4.2: Django~=4.2
+    django5.1: Django~=5.1
+    django5.2: Django~=5.2
```

Comment:
These version qualifiers are effectively equivalent - they will select the latest patch release available for the specified range, but it's a bit shorter and easier to maintain.